### PR TITLE
unify font sizes

### PR DESCRIPTION
**Pull request by elharo**
@chanseokoh  @briandealwis

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/AnalyticsOptInArea.java [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 import org.eclipse.core.runtime.IStatus;
 import org.eclipse.core.runtime.Status;
+import org.eclipse.jface.dialogs.Dialog;
```
Comment (chanseokoh) [inline plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/AnalyticsOptInArea.java]: This is not used.

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/AnalyticsOptInArea.java]: removed

**Comment by chanseokoh** [comment]
Looks good. I just wonder why this isn't happening on my Ubuntu and why the fonts of the two controls are different. I'll let @briandealwis to give LGTM.

**Comment by briandealwis** [comment]
Kinda fascinating.  `Link` widgets on OS X have a smaller default font of 12 points vs the default for other widgets of 13 points.  This is the same approach taken with the JFace `FieldEditor`s, so LGTM.

**Comment by briandealwis** [comment]
#424

**Pull request merged**